pkg/manifestclient/testing/create_test.go

```diff
@@ -0,0 +1,146 @@
+package testing
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"path/filepath"
+	"regexp"
+	"testing"
+
+	"github.com/openshift/library-go/pkg/manifestclient"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/errors"
+	applyconfigurationscorev1 "k8s.io/client-go/applyconfigurations/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+func TestUpdateConfigMapLabel(t *testing.T) {
+	client := setupKubernetesClient(t)
+	configMapList, err := client.CoreV1().ConfigMaps("openshift-oauth-apiserver").List(context.TODO(), metav1.ListOptions{})
```

Comment:
You need to change the namespace to "openshift-authentication", there are no configMaps in this namespace